panther_bringup/launch/bringup.launch.py

```diff
@@ -39,7 +39,7 @@ def generate_launch_description():
     namespace = LaunchConfiguration("namespace")
     declare_namespace_arg = DeclareLaunchArgument(
         "namespace",
-        default_value=EnvironmentVariable("ROBOT_NAMESPACE", default_value=""),
+        default_value=EnvironmentVariable("ROBOT_NAMESPACE", default_value="panther"),
```

Comment:
If we change the default value we should remember to update main compose, but I don't think this lays in scope of this task. Maybe you could create reminder somewhere?

Reply:
In compose is `namespce:=panther` so I belived it should be also changed here.

Reply:
I agree, I think the same before first release 2.0.0.

Reply:
Ok, so lets just change this in compose :P